coinjoin/coinjoin.go

```diff
@@ -229,11 +229,17 @@ func (t *Tx) Join(unmixed []byte, pid int) error {
 	if other.Version != t.txVersion {
 		return errors.New("coinjoin: different tx versions")
 	}
+	if len(other.TxOut) == 0 {
```

Comment:
I thought it was implicitly implied, but this and the other change in this file are also unnecessary for the same reasons as in the original review.